tasks/check_mounts.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Ensure correct permissions are set for the mountpoint
+  ansible.builtin.file:
+    path: "{{ EXTERNAL_STORAGE_DRIVE }}"
```

Comment:
There doesn't seem to be a default for `EXTERNAL_STORAGE_DRIVE`. The README.md says it is `/storage` but that is only set for Molecule.

Reply:
the tests weren't running because the playbook was only targeting the `db` hosts rather than `all` (there's no `db` host in this setup, must be a leftover from copying the `postgresql` role setup). I think it's running correctly now